[2023-11-24T22:22:12.903+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T19:29:07.263785+00:00 [queued]>
[2023-11-24T22:22:12.917+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T19:29:07.263785+00:00 [queued]>
[2023-11-24T22:22:12.918+0000] {taskinstance.py:1361} INFO - Starting attempt 22 of 22
[2023-11-24T22:22:12.935+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Data_transform_and_load> on 2023-11-22 19:29:07.263785+00:00
[2023-11-24T22:22:12.942+0000] {standard_task_runner.py:57} INFO - Started process 4477 to run task
[2023-11-24T22:22:12.946+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'COVID-19', 'Data_transform_and_load', 'manual__2023-11-22T19:29:07.263785+00:00', '--job-id', '104', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl.py', '--cfg-path', '/tmp/tmpg_21ogr7']
[2023-11-24T22:22:12.949+0000] {standard_task_runner.py:85} INFO - Job 104: Subtask Data_transform_and_load
[2023-11-24T22:22:13.004+0000] {task_command.py:416} INFO - Running <TaskInstance: COVID-19.Data_transform_and_load manual__2023-11-22T19:29:07.263785+00:00 [running]> on host 3742b108fdc7
[2023-11-24T22:22:13.255+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='COVID-19' AIRFLOW_CTX_TASK_ID='Data_transform_and_load' AIRFLOW_CTX_EXECUTION_DATE='2023-11-22T19:29:07.263785+00:00' AIRFLOW_CTX_TRY_NUMBER='22' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-22T19:29:07.263785+00:00'
[2023-11-24T22:22:13.821+0000] {base.py:73} INFO - Using connection ID 'postgres_connexion' for task execution.
[2023-11-24T22:53:56.766+0000] {python.py:194} INFO - Done. Returned value was: Data inserted successfully
[2023-11-24T22:53:56.817+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=COVID-19, task_id=Data_transform_and_load, execution_date=20231122T192907, start_date=20231124T222212, end_date=20231124T225356
[2023-11-24T22:53:56.884+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-11-24T22:53:56.903+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
